test/unit/resources/ssl_test.rb

```diff
@@ -0,0 +1,57 @@
+# encoding: utf-8
+# author: Christoph Hartmann
+# author: Dominik Richter
```

Comment:
Feel free to remove our names from the authors list :)

Reply:
Sounds good, just pushed out a change to remove them.